product.html

```diff
@@ -141,5 +144,200 @@
+                      fill="#363535" />
+                  </g>
+                  <defs>
+                    <clipPath id="clip0_2923_446">
+                      <rect width="16" height="16" fill="white" />
+                    </clipPath>
+                  </defs>
+                </svg> Retiro en Tienda
+              </button>
+              </div>
+            </form>
+          </div>
+          <form action="" method="post" class="product__add-card">
+            <div class="product__add-card-counter">
+              <button onclick="restar()" type="button" class="product__add-card-counter-button product__add-card-counter-button--less-add">
```

Comment:
Preferiblemente no uses onclick u otro evento de javascript en el html, es mejor asignarlo en el archivo .js para tener las cosas mas ordenadas
```suggestion
              <button type="button" class="product__add-card-counter-button product__add-card-counter-button--less-add js-substract-unit-to-product">
```